packages/devops/scripts/lambda/helpers/teardown.py

```diff
@@ -57,3 +58,15 @@ def teardown_instance(instance):
       delete_gateway(deployment_type, deployment_name)
 
     terminate_instance(instance)
+
+def teardown_db_instance(
+  instance_name
+):
+    deleted_instance = rds.delete_db_instance(
+        DBInstanceIdentifier=instance_name,
```

Comment:
I reckon we should make the identifier either `deployment_type: instance_name` like on the EC2 instances, or look up the db instance by tags here (including deployment type)

Reply:
So it'd be like `tupaia: rn-195_move_to_rds`? I'm down with that 👍

Reply:
So due to restrictions in the naming convention for RDS databases, it's gonna have to be `tupaia-<deployment-name>`
https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Limits.html